lib/liftoff/cli.rb

```diff
@@ -7,7 +7,7 @@ def initialize(argv)
 
     def run
       parse_command_line_options
-      LaunchPad.new.liftoff @options
+      LaunchPad.new.liftoff @options, @argv.first
```

Comment:
I was reluctant about that. It feels weird because it could mean that you can preset the path from the config file.